### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by wvankuipers**
# Contribution description
Implementing Cucumber 3

# Pull request checklist
- [x] Contributed code respects the [editorconfig rules](.editorconfig)
- [x] Contributed code passes the [eslint rules](.eslintrc.yaml)
- [x] Contributed code passes the unit tests
- [ ] Added rules are described in the [readme file](README.md)
- [x] [The build](https://travis-ci.org/webdriverio/cucumber-boilerplate) of the PR is passing

### CODE REVIEW

**Comment by wvankuipers** [comment]
There are some tests failing due to limitations in the latest version of PhantomJS (I think)
Will investigate and come up with a solution for those tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.3.0] - T.B.D.
+### Added
+- Cucumber 3 as a dependency
```
Comment (christian-bromann) [inline CHANGELOG.md]: afaik WebdriverIO doesn't support cucumber v3 yet when running it via wdio-cucumber-framework

File: package.json [hunk 2]
```diff
@@ -32,18 +32,20 @@
   "dependencies": {
     "babel-preset-es2015": "~6.24.0",
     "babel-register": "~6.26.0",
-    "chai": "~4.1.0",
+    "chai": "~4.1.2",
+    "cucumber": "~3.0.1",
```
Comment (christian-bromann) [inline package.json]: why do you have cucumber as dependency defined here? It is already a dependency of wdio-cucumber-framework.

**Comment by christian-bromann** [comment]
Congratz @wvankuipers 👏

**Pull request merged**